dali/python/nvidia/dali/ops.py

```diff
@@ -333,3 +334,46 @@ def __call__(self, *inputs, **kwargs):
         op_instance.spec.AddArg("feature_names", feature_names)
         op_instance.spec.AddArg("features", features)
         return outputs
+
+
+class PythonFunction(with_metaclass(_DaliOperatorMeta, object)):
+    def __init__(self, function, **kwargs):
+        self._schema = b.GetSchema("PythonFunctionImpl")
```

Comment:
Please check if you can load the plugin here. It'd save some startup time (and memory).